src/librustc/hir/map/mod.rs

```diff
@@ -159,6 +159,10 @@ impl Forest {
         self.dep_graph.read(DepNode::new_no_params(DepKind::Krate));
         &self.krate
     }
+
+    pub fn untracked_krate<'hir>(&'hir self) -> &'hir Crate {
```

Comment:
Please add a warning comment here not to use this function and that it only exists for internal use with the dep-tracking system.